src/templates/layout.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+  <!-- <<<<<<< HEAD  -->
+
+  <head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="copyright" content="Copyright 2016 all rights reserved by Jan Werkhoven">
+    <meta name="developer" content="Jan Werkhoven jan@nabu.io www.nabu.io">
+    <meta name="robots" content="index, follow">
+
+    <meta name="google-site-verification" content="lZD8Pa_Me8epqXpYFLBunfhCMuWaA02nhzvzRpRvlRc" />
+    {{#block "meta"}}
+      <title>FinSec PTX</title>
+      <meta name="description" content="Experts in transfering UK pensions to Australian superannuations"> {{/block}}
+
+    <link href="/assets/css/main.min.css" rel="stylesheet" type="text/css">
+
+    <script src="/assets/js/outdated-browser.js"></script>
+    <script src="/assets/js/main.js"></script>
+
+    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="/assets/img/favicons/apple-touch-icon-57x57.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/assets/img/favicons/apple-touch-icon-114x114.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/assets/img/favicons/apple-touch-icon-72x72.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/assets/img/favicons/apple-touch-icon-144x144.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/assets/img/favicons/apple-touch-icon-120x120.png" />
+    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/assets/img/favicons/apple-touch-icon-152x152.png" />
+    <link rel="icon" type="image/png" href="/assets/img/favicons/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="/assets/img/favicons/favicon-16x16.png" sizes="16x16" />
+    <meta name="application-name" content="&nbsp;" />
+    <meta name="msapplication-TileColor" content="#FFFFFF" />
+    <meta name="msapplication-TileImage" content="/assets/img/favicons/mstile-144x144.png" />
+
+  </head>
+
+  <body id="{{#block "body-id"}}{{/block}}" class="{{#block "body-class"}}{{/block}}">
+
+    {{> header}}
+    <main>
+      {{#block "main"}}{{/block}}
+    </main>
+    {{> footer}}
+
+    <div id="outdated"></div>
+    <script src="/assets/js/outdated-browser.js" type="text/javascript"></script>
+
+  </body>
+  <!-- =======
```

Comment:
If I were you, I would reset this one file. This is to important. You can pull in only this one file from `staging` by bashing `git checkout staging src/templates/layout.html` in your repo root. Then add, commit en push.